packages/engine/src/gltf/SceneDeltaExporterExtension.ts

```diff
@@ -0,0 +1,65 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/ir-engine/ir-engine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Infinite Reality Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Infinite Reality Engine team.
+
+All portions of the code written by the Infinite Reality Engine team are Copyright © 2021-2023 
+Infinite Reality Engine. All Rights Reserved.
+*/
+
+import { getComponent, iterateEntityNode } from '@ir-engine/ecs'
+import { getState } from '@ir-engine/hyperflux'
+import { cleanStorageProviderURLs } from '../assets/functions/parseSceneJSON'
+import { SourceComponent } from '../scene/components/SourceComponent'
+import { SceneDeltaRegistry, SceneDeltaState } from '../scene/systems/SceneDeltaState'
+import { GLTFSceneExportExtension } from './exportGLTFScene'
+import { GLTFComponent } from './GLTFComponent'
+import { NodeIDComponent } from './NodeIDComponent'
+
+export const SCENE_DELTA_EXTENSION_NAME = 'IR_scene_delta'
```

Comment:
do we want this to be called a 'scene delta' or is a 'delta' or something else more appropriate?

Reply:
I think scene delta for now because they are necessarily tied to the GLTF scene spec. This implementation is for top level deltas that all get saved directly into the root of the scene, rather than partials that get attached to nodes individually